cli/src/factory_ops/unboxing/init

```diff
@@ -336,33 +336,12 @@ echo "Removing the repos with Seagate URL from Server A" | tee -a ${LOG_FILE}
 seagate_refs_cleanup
 
 echo "Restarting the cluster resources.........." | tee -a ${LOG_FILE}
-echo -n "ClusterIP.............................." | tee -a ${LOG_FILE}
```

Comment:
Instead of deleting, guess we should check feasibility of moving this block to well-defined function lets say **restart_cluster_resources** (Just for backward-compatibility support even though we will not be calling this function).

This comment need not be addressed immediately in-case we need urgent fix (but should be planned for LR2 later).